admin/views/accessibility-statement.ejs

```diff
@@ -95,9 +95,8 @@
                 </tr>
                 <tr>
                     <td>Audio version</td>
-                    <td>Not recommended for iOS 9.3 Not compatible with any
-                    version of Internet Explorer and below version 14 of
-                    Microsoft Edge
+                    <td>Not recommended for iOS and Safari below version 12,
```

Comment:
If this is for the Pupil SPA then we need iOS / Safari >= 13

Reply:
this needs raising internally, copied verbatim

Reply:
changing to 13